src/main/java/belly/Belly.java

```diff
@@ -0,0 +1,91 @@
+package belly;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+import duke.DukeException;
+import processor.TaskDatetimeFormatter;
+import task.Deadline;
+import task.Event;
+import task.Task;
+import task.Todo;
+
+/**
+ * Represents a storage system to load data from or save data to local hard disk.
+ */
+public class Belly {
+    public static final String FILE_DIRECTORY = "../data/";
```

Comment:
Does this work on all platforms Windows, Linux, etc?
You can try using the relative home directory.
```java
public Belly(String folderName) throws DukeException {
    String userHome = System.getProperty("user.home");
    this.storagePath = Paths.get(userHome, ".duke", "belly", folderName);
    try {
        Files.createDirectories(storagePath);
    } catch (UnsupportedOperationException | SecurityException | IOException e) {
        throw new DukeException(e.getMessage());
    }
}
```